newm-server/src/main/kotlin/io/newm/server/health/HealthCheckInstall.kt

```diff
@@ -0,0 +1,41 @@
+package io.newm.server.health
+
+import cc.rbbl.ktor_health_check.Health
+import com.amazonaws.services.s3.AmazonS3
+import com.zaxxer.hikari.HikariDataSource
+import io.ktor.server.application.Application
+import io.ktor.server.application.ApplicationEnvironment
+import io.ktor.server.application.install
+import io.newm.shared.koin.inject
+import io.newm.shared.ktx.getConfigString
+
+// livez is the preferred endpoint over the deprecated healthz. We implement both for good measure.
+private const val HEALTHZ_CHECK_URL = "healthz"
+private const val LIVEZ_CHECK_URL = "livez"
+
+fun Health.Configuration.healthtechChecks() {
+    listOf(HEALTHZ_CHECK_URL, LIVEZ_CHECK_URL).forEach { endpoint ->
```

Comment:
Maybe the Url list can come from application.conf and retrieved using `environment .getConfigStrings`